#!/usr/bin/env python
PACKAGE = "projectbergamascodigiusto"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("x_initial",    int_t,    0, "An Integer parameter", 0,  -50, 50)
gen.add("y_initial",    int_t,    0, "An Integer parameter", 0,  -50, 50)

odom_enum = gen.enum([ gen.const("Differential",     int_t, 1,"Differential"),
                       gen.const("Ackerman",     int_t, 2,"Ackerman")],
                     "An enum to set the type of Odometry")

gen.add("odometry_type", int_t, 4, "Choose type of Odometry to be used", 1, 1, 2, edit_method=odom_enum)

exit(gen.generate(PACKAGE, "dynamic_rcfg", "dynamic"))
